### PR TITLE
Add scope documentation

### PR DESCRIPTION
**Pull request by spheric**

### CODE REVIEW

File: source/includes/_users.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ authentication[access_token] | string | The access token to include in authorize
 authentication[token_type] | string | Always `Bearer`
 authentication[expires_in] | integer | The number of seconds the access token is valid for
 authentication[refresh_token] | string | The refresh token which is exchanged for a new access token
-authentication[scope] | string | The scopes, either `[ read, write ]`, the access token is/are valid for
+authentication[scope] | string | The scopes, one of scopes listed under authentication
```
Comment (adriancb) [inline source/includes/_users.md]: One or more I'd say.

File: source/index.html.md [hunk 2]
```diff
@@ -98,11 +99,32 @@ The endpoint accepts one parameter and a [Basic authentication](https://en.wikip
...
+
+Name | Description
+--------- | -----------
+questionnaire.questions.read | Read questionnaire question data
+questionnaire.results.read | Read questionnaire result data
+users.read | Read user data
+users.write | Write user data
+users.accounts.read | Read user accounts data
+users.accounts.write | Write user accounts data
+users.accounts.results.read | Read user accounts result data
```
Comment (adriancb) [inline source/index.html.md]: I made a comment about this one in the other PR.

File: source/index.html.md [hunk 3]
```diff
@@ -98,11 +99,32 @@ The endpoint accepts one parameter and a [Basic authentication](https://en.wikip
...
+users.accounts.portfolio.read | Read user accounts portfolio data
+
 ### FORM Parameters
 
 Parameter | Required | Type | Default | Description
 --------- | ----------- | ----------- | ----------- | -----------
 grant_type | yes | string | `no default` | Must be `client_credentials`
+scope | no | string | `no default` | Must be one or many scopes separated by a ' '
+
+### Query Parameters
```
Comment (adriancb) [inline source/index.html.md:122]: I don't believe this is required as you can use either a `GET` or `POST` with form parameters. Thoughts?

File: source/index.html.md (hunk 2 above)
Comment (spheric) [reply source/index.html.md]: oh yep that's right and I renamed it lol will fix

File: source/index.html.md (hunk 3 above)
Comment (spheric) [reply source/index.html.md:122]: yeah just thought I would supply as an option both work but happy to just force one way

Comment (spheric) [reply source/index.html.md:122]: maybe make it obvious that it's either form or query?

**Pull request merged**